starter-code/memory.js

```diff
@@ -121,19 +122,21 @@ function eventListener(globalCounter) {
         choices[1].parents().addClass("complete");
         // checkCompletion();
         globalCounter += 2; // Counter to keep track of how many completed
-        if (globalCounter == 24) { // If counter == to 24 then game complete
+        globalCounter = parseInt(globalCounter);
+        console.log(globalCounter);
+        if (globalCounter === 24) { // If counter == to 24 then game complete
           $("#overlay").show();
           $("#overlay").delay(200).fadeOut(100).delay(200).fadeIn(100).delay(200).fadeOut(100).delay(200).fadeIn(100);
           $('#restart').click(function (e) {
             location.reload(true);
         }); } else {
-          eventListener();
+          eventListener(globalCounter);
         }
       } else {
         setTimeout(function(){
           choices[0].hide();
           choices[1].hide();
-          eventListener();
+          eventListener(globalCounter);
         }, 1500);
 
       }
```

Comment:
good job! If you have you could try to refactor the javascript part and get a cleaner view of the grid